@types/berlin-de-article.ts

```diff
@@ -0,0 +1,4 @@
+type BerlinDEArticle = {
```

Comment:
I'm not sure about this, but as I can see here, this is a global module, isn't it? Probably it was defined on tsconfig, but to make this file more meaningful, use the suffix `d.ts` like cheerio, then it will indicate that it is a global definition.